tests/tests.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Unit test suite
+ * Prerequisites: nodeunit ("npm install nodeunit -g")
+ * Help: https://github.com/caolan/nodeunit
+ * Some tests involves connecting to a bogus database. Therefore, you might need to run "rethinkdb".
+ */
+process.chdir(__dirname);
+
+try {
+  var reporter = require('nodeunit').reporters.default;
```

Comment:
I'm going to check out if some of the other non-`default` reporters (https://www.npmjs.com/package/nodeunit#built-in-test-reporters) are better for running bulk tests.